docs/excel/excel-add-ins-troubleshooting.md

```diff
@@ -41,6 +41,56 @@ The following APIs are affected by this workbook switch:
 
 See [Coauthoring in Excel add-ins](co-authoring-in-excel-add-ins.md) for patterns to use with events in a coauthoring environment. The article also discusses potential merge conflicts when using certain APIs, such as [`TableRowCollection.add`](/javascript/api/excel/excel.tablerowcollection#add-index--values-).
 
+## Known Issues
+
+### Binding events ID discrepancy
+
+Both [BindingDataChangedEventArgs.binding](/javascript/api/excel/excel.bindingdatachangedeventargs#binding) and [BindingSelectionChangedEventArgs.binding](/javascript/api/excel/excel.bindingselectionchangedeventargs#binding) return a temporary `Binding` object that contains the ID of the `Binding` object that raised the event. Use this ID with `BindingCollection.getItem(id)` to retrieve the `Binding` object that raised the event.
```

Comment:
Minor suggestion to avoid repetition.
```suggestion
Both [BindingDataChangedEventArgs.binding](/javascript/api/excel/excel.bindingdatachangedeventargs#binding) and [BindingSelectionChangedEventArgs.binding](/javascript/api/excel/excel.bindingselectionchangedeventargs#binding) return a temporary `Binding` object that contains the ID of the `Binding` object that raised the event. Use this ID with `BindingCollection.getItem(id)` to retrieve the relevant `Binding` object.
```

Reply:
I agree that this sentence is repetitive (and I actually had it phrased exactly like your suggestion at one point), but I think the explicit repetition is helpful because there are technically two relevant `Binding` objects. It's such a confusing concept that I want to be ax explicit as possible.